index.html

```diff
@@ -0,0 +1,184 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./css/tablet.css">
+    <link rel="stylesheet" href="./css/mobile.css">
+    <title>SINGOLO</title>
+</head>
+<body>
+    <header class="header wrapper">
+        <button class="burger-menu">
+            <svg width="25" height="20" viewBox="0 0 25 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1.2C0 0.537259 0.537259 0 1.2 0H23.8C24.4627 0 25 0.537259 25 1.2C25 1.86274 24.4627 2.4 23.8 2.4H1.2C0.537259 2.4 0 1.86274 0 1.2ZM0 10C0 9.33726 0.537259 8.8 1.2 8.8H23.8C24.4627 8.8 25 9.33726 25 10C25 10.6627 24.4627 11.2 23.8 11.2H1.2C0.537259 11.2 0 10.6627 0 10ZM1.2 17.6C0.53726 17.6 0 18.1373 0 18.8C0 19.4627 0.537259 20 1.2 20H23.8C24.4627 20 25 19.4627 25 18.8C25 18.1373 24.4627 17.6 23.8 17.6H1.2Z" fill="white"/>
+            </svg>
+
+        </button>
+        <a class="logo" href="#">
+            <img class="logo__component" src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="header-nav">
+            <ul class="header-nav-list">
+                <li class="header-nav-list__component"><a class="header-nav-list__component-link" href="#">HOME</a></li>
+                <li class="header-nav-list__component"><a class="header-nav-list__component-link" href="#">SERVICES</a></li>
+                <li class="header-nav-list__component"><a class="header-nav-list__component-link" href="#">PORTFOLIO</a></li>
+                <li class="header-nav-list__component"><a class="header-nav-list__component-link" href="#">ABOUT</a></li>
+                <li class="header-nav-list__component"><a class="header-nav-list__component-link" href="#">CONTACT</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="content wrapper">
+        <section class="slider slider-1">
+            <div class="slider-nav-block slider-nav-block-margin">
+                <button class="slider-nav-block__component slider-button-prev">
+                    <svg class="slider-button-prev" width="16" height="27" viewBox="0 0 16 27" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M0.697849 22.9923C-0.233349 23.9092 -0.232514 25.3553 0.699743 26.2713C1.68867
+                     27.2429 3.33657 27.2429 4.32549 26.2713L15.0449 15.7392C16.3184 14.4879 16.3184 12.5121
+                      15.0449 11.2608L4.32549 0.728735C3.33657 -0.242913 1.68867 -0.242911 0.699746
+                       0.728737C-0.232511 1.64471 -0.233347 3.09077 0.69785 4.00769L8.06645 11.2634C9.33653
+                        12.514 9.33653 14.486 8.06645 15.7366L0.697849 22.9923Z" fill="#E94348"/>
+                    </svg>
+                </button>
+                <button class="slider-nav-block__component slider-button-next">
```

Comment:
You don't need 2 icons here. You need only one and for the second arrow you can make this hack "transform: rotate(180deg);"